crates/holochain_conductor_services/src/dpki_service.rs

```diff
@@ -48,8 +48,10 @@ impl DpkiService {
             state,
         }
     }
-    pub fn should_run(&self, dna_hash: &DnaHash) -> bool {
-        self.cell_id.dna_hash() != dna_hash
+
+    /// Whether the passed in DNA hash is Deepkey DNA hash
+    pub fn is_deepkey_dna(&self, dna_hash: &DnaHash) -> bool {
```

Comment:
I was confused by `should_run` and thought first it's something like `ensure` for the dpki service. Then I saw that it's used to check whether sys validation should run for Deepkey ops.
I suggest to rename to either the above or `should_run_validation`.